db/seeds.rb

```diff
@@ -5,3 +5,14 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+require 'random_data'
+require 'faker'
+
+10.times do
+    Resource.create!(
+      title:  Faker::Book.title,
+      resource_type:   RandomData.random_resource_type,
+      vendor: Faker::Company.title
+    )
+  end
+  resources = Resource.all
```

Comment:
don't need this since you don't do anything with it

Reply:
yeah, was going to output the total at the bottom...i get a little ahead of myself sometimes